services/docker-compose.yml

```diff
@@ -148,11 +148,36 @@ services:
         - traefik.http.services.${SWARM_STACK_NAME}_static_webserver.loadbalancer.healthcheck.timeout=1000ms
         - traefik.http.middlewares.${SWARM_STACK_NAME}_static_webserver_retry.retry.attempts=2
         - traefik.http.routers.${SWARM_STACK_NAME}_static_webserver.rule=hostregexp(`{host:.+}`)
-          && (PathPrefix(`/osparc`) || PathPrefix(`/s4l`) || PathPrefix(`/tis`)
-          || PathPrefix(`/transpiled`) || PathPrefix(`/resource`))
+          && (PathPrefix(`/osparc`,`/s4l`,`/tis`,`/transpiled`,`/resource`))
         - traefik.http.routers.${SWARM_STACK_NAME}_static_webserver.entrypoints=http
         - traefik.http.routers.${SWARM_STACK_NAME}_static_webserver.priority=2
         - traefik.http.routers.${SWARM_STACK_NAME}_static_webserver.middlewares=${SWARM_STACK_NAME}_gzip@docker,${SWARM_STACK_NAME}_static_webserver_retry
+        # catchall for legacy services (this happens if a backend disappears and a frontend tries to reconnect, the right return value is a 503)
+        - traefik.http.routers.${SWARM_STACK_NAME}_legacy_services_catchall.service=${SWARM_STACK_NAME}_legacy_services_catchall
+        - traefik.http.routers.${SWARM_STACK_NAME}_legacy_services_catchall.priority=1
+        - traefik.http.routers.${SWARM_STACK_NAME}_legacy_services_catchall.entrypoints=http
+        - traefik.http.routers.${SWARM_STACK_NAME}_legacy_services_catchall.rule=hostregexp(`{host:.+}`)
+          && PathPrefix(`/x`)
+        # this tricks traefik into a 502 since the service does not exist on this port
```

Comment:
MINOR: i sometimes get them wrong, can we also add the code's names?

e.g.
```
https://httpstatuses.com/502 - bad gateway
https://httpstatuses.com/503 - service unavailable
```

Reply:
ok